### PR TITLE
레지스터 페이지 작업, 1차입니다.

### PR DESCRIPTION
**Pull request by chepchep2**

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#17** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/17?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#16** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/16?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#15** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/15?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#14** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/14?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#13** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/13?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/12?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#11** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/11?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#10** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/10?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#9** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/9?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#8** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/8?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/7?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#6** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/6?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/5?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/4?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/3?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/2?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/common/auth_text_input_widget.dart [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    required this.labelText,
+    required this.hintText,
+  });
+
+  @override
+  State<AuthTextInputWidget> createState() => _AuthTextInputWidgetState();
+}
+
+class _AuthTextInputWidgetState extends State<AuthTextInputWidget> {
+  final TextEditingController _controller = TextEditingController();
```
Comment (flowkater) [inline lib/common/auth_text_input_widget.dart:18]: TextEditingController 도 _isTextNotEmpty 도 State 를 밖에서 주입을 받으면 좋을 것 같고, 해당 Widget 은 Stateless 하게 유지하면 좋을 것 같습니다.

File: lib/common/auth_text_input_widget.dart [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+              hintText: widget.hintText,
+              hintStyle: const TextStyle(
+                fontSize: 20,
+                color: Colors.grey,
+                fontWeight: FontWeight.bold,
+              ),
+              suffixIcon: IconButton(
+                icon: const Icon(Icons.cancel),
+                color: _isTextNotEmpty ? Colors.black : Colors.grey,
+                onPressed: () {
```
Comment (flowkater) [inline lib/common/auth_text_input_widget.dart:56]: 이것도 함수를 주입 받으면 좋을 것 같습니다. 직접 setState 를 호출하기보다

**Review by flowkater — CHANGES_REQUESTED** [review]
위 상황 한번 반영 부탁드립니다.

File: lib/common/auth_text_input_widget.dart (hunk 1 above)
Comment (chepchep2) [reply lib/common/auth_text_input_widget.dart:18]: 네 다시 해보겠습니다.

File: lib/common/auth_text_input_widget.dart (hunk 2 above)
Comment (chepchep2) [reply lib/common/auth_text_input_widget.dart:56]: 다시 해보겠습니다.

**Comment by flowkater** [comment]
### Merge activity

* **Mar 7, 8:27 AM EST**: @flowkater started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/3).
* **Mar 7, 8:30 AM EST**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/3) rebased this pull request as part of a merge.
* **Mar 7, 8:31 AM EST**: @flowkater merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/3).

**Review by flowkater — APPROVED** [review]

**Pull request merged**